packages/synthetix-main/contracts/interfaces/IAccountModule.sol

```diff
@@ -89,4 +98,9 @@ interface IAccountModule {
      * @dev Returns the address for the account token used by the module.
      */
     function getAccountTokenAddress() external view returns (address);
+
+    /**
+     * @dev Returns the address that owns a given account, as recorded by the system.
+     */
+    function accountOwner(uint accountId) external view returns (address);
```

Comment:
Added this function to have visibility on who the core system caches to be the owner of an account.